src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,725 @@
+INTACT_INCREASES_ACTIONS = {
+    'psi-mi:"MI:1143"(aminoacylation reaction)',
+    'psi-mi:"MI:0214"(myristoylation reaction)',
+    'psi-mi:"MI:1237"(proline isomerization reaction)',
+    'psi-mi:"MI:0211"(lipid addition)',
+    'psi-mi:"MI:0213"(methylation reaction)',
+    'psi-mi:"MI:0217"(phosphorylation reaction)',
+    'psi-mi:"MI:0882"(atpase reaction)',
+    'psi-mi:"MI:1250"(isomerase reaction)',
+    'psi-mi:"MI:0210"(hydroxylation reaction)',
+    'psi-mi:"MI:0883"(gtpase reaction)',
+    'psi-mi:"MI:0557"(adp ribosylation reaction)',
+    'psi-mi:"MI:0193"(amidation reaction)',
+    'psi-mi:"MI:1327"(sulfurtransfer reaction)',
+    'psi-mi:"MI:0567"(neddylation reaction)',
+    'psi-mi:"MI:0556"(transglutamination reaction)',
+    'psi-mi:"MI:0220"(ubiquitination reaction)',
+    'psi-mi:"MI:0559"(glycosylation reaction)',
+    'psi-mi:"MI:0192"(acetylation reaction)',
+    'psi-mi:"MI:0216"(palmitoylation reaction)',
+    'psi-mi:"MI:0945"(oxidoreductase activity electron transfer reaction)',
+    'psi-mi:"MI:0701"(dna strand elongation)',
+    'psi-mi:"MI:0844"(phosphotransfer reaction)',
+    'psi-mi:"MI:1148"(ampylation reaction)',
+    'psi-mi:"MI:0566"(sumoylation reaction)',
+    'psi-mi:"MI:1146"(phospholipase reaction)',
+}
+
+#: Relationship types in IntAct that map to BEL relation 'decreases'
+INTACT_DECREASES_ACTIONS = {
+    # decreases
+    'psi-mi:"MI:0902"(rna cleavage)',
+    'psi-mi:"MI:0572"(dna cleavage)',
+    'psi-mi:"MI:0199"(deformylation reaction)',
+    'psi-mi:"MI:2280"(deamidation reaction)',
+    'psi-mi:"MI:0203"(dephosphorylation reaction)',
+    'psi-mi:"MI:0212"(lipoprotein cleavage reaction)',
+    'psi-mi:"MI:0570"(protein cleavage)',
+    'psi-mi:"MI:0204"(deubiquitination reaction)',
+    'psi-mi:"MI:0871"(demethylation reaction)',
+    'psi-mi:"MI:0985"(deamination reaction)',
+    'psi-mi:"MI:1355"(lipid cleavage)',
+    'psi-mi:"MI:0569"(deneddylation reaction)',
+    'psi-mi:"MI:1140"(decarboxylation reaction)',
+    'psi-mi:"MI:0194"(cleavage reaction)',
+    'psi-mi:"MI:0197"(deacetylation reaction)',
+}
+
+#: Relationship types in IntAct that map to BEL relation 'association'
+INTACT_ASSOCIATION_ACTIONS = {
+    'psi-mi:"MI:1127"(putative self interaction)',
+    'psi-mi:"MI:0914"(association)',
+    'psi-mi:"MI:1126"(self interaction)',
+    'psi-mi:"MI:0915"(physical association)',
+    'psi-mi:"MI:0414"(enzymatic reaction)',
+    'psi-mi:"MI:0403"(colocalization)',
+}
+
+#: Relationship types in IntAct that map to BEL relation 'regulates'
+INTACT_REGULATES_ACTIONS = {
+    'psi-mi:"MI:0407"(direct interaction)',
+}
+
+#: Relationship types in IntAct that map to BEL relation 'hasComponent'
+INTACT_BINDS_ACTIONS = {
+    'psi-mi:"MI:0195"(covalent binding)',
+    'psi-mi:"MI:0408"(disulfide bond)',
+}
+
+SUBJECT_ACTIVITIES = {
+    'psi-mi:"MI:0883"(gtpase reaction)': pybel.dsl.activity(
+        name='GTPase activity',
+        namespace='GO',
```

Comment:
all namespaces should be lowercased

Reply:
That's how you added them in the first place, so I adapted to that. But I can change it